build.gradle

```diff
@@ -48,6 +48,9 @@ dependencies {
     implementation("org.springframework.boot:spring-boot-starter-data-jpa")
     implementation("org.springframework.boot:spring-boot-starter-jdbc")
     implementation("org.postgresql:postgresql")
+    implementation("io.github.cdimascio:dotenv-kotlin:6.3.1")
```

Comment:
A Springboot application already has an environment file in terms of the properties file (e.g., src/main/resources/application.properties).

Using multiple environment files is, then, redundant and confusing.

Reply:
Removed
Added API key onto application.properties instead